presto-main/src/main/java/com/facebook/presto/catalogserver/RemoteMetadataManager.java

```diff
@@ -194,4 +322,79 @@ private <T> T readValue(String content, TypeReference<T> valueTypeRef)
             throw new RuntimeException(e);
         }
     }
+
+    private static void incrementCacheHitCount(Session session, boolean isCacheHit)
```

Comment:
Why these methods are `static`?

Reply:
I thought it would be better to use them as static when they dont make use of any instance fields, can make all helper methods non static though, done